index.js

```diff
@@ -199,13 +199,13 @@ var Base = Class.extend({
     }
 
     var pkSql = '';
-    if (primaryKeyColumns.length > 1) {
-      pkSql = this._handleMultiPrimaryKeys(primaryKeyColumns);
-
-    } else {
-
-      primaryKeyColumns[0] = primaryKeyColumns[0].name;
-      columnDefOptions.emitPrimaryKey = true;
+    if (primaryKeyColumns.length) {
```

Comment:
How is this supposed to solve a problem?
https://github.com/db-migrate/db-migrate-base/blob/master/index.js#L188

As you can see primaryKeyColumns is never undefined though.